diff -burN ossim-1.8.20-3.orig/ossim_package_support/cmake/CMakeModules/FindTIFF.cmake ossim-1.8.20-3/ossim_package_support/cmake/CMakeModules/FindTIFF.cmake
--- ossim-1.8.20-3.orig/ossim_package_support/cmake/CMakeModules/FindTIFF.cmake	2015-01-20 16:20:12.000000000 +0000
+++ ossim-1.8.20-3/ossim_package_support/cmake/CMakeModules/FindTIFF.cmake	2019-05-09 11:49:28.258565000 +0000
@@ -21,6 +21,8 @@
 #  License text for the above reference.)
 
 FIND_PATH(TIFF_INCLUDE_DIR tiff.h
+          PATH_SUFFIXES
+            include
           PATHS 
           ${CMAKE_INSTALL_PREFIX}/include
           /usr/local/include 
@@ -34,6 +36,8 @@
 # Added x86_64-linux-gnu path for Ubuntu install
 FIND_LIBRARY(TIFF_LIBRARY 
              NAMES ${TIFF_NAMES} 
+             PATH_SUFFIXES
+               lib
              PATHS 
              ${CMAKE_INSTALL_PREFIX}/lib
              /usr/local/lib
